Add UI feedback when there's no followed players to warp to as ghost

qwerltaz opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
~~Add popup~~ grey out the "warp to most followed" button when there's no followed players as ghost.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
zero feedback on button press no good.

qwerltaz committed:
init

IProduceWidgets commented:
Or just have it select a random player to follow

iaada commented:
Disabling the button and changing the text feels right. I would like to see a "warp to random player" button on its own

qwerltaz committed:
Revert "init"

This reverts commit 1c3d86d346e44b3816fd433eb0361218f53ba22d.

qwerltaz committed:
add periodic update

qwerltaz committed:
disable button when nobody being followed

qwerltaz committed:
dont refresh entire warp list

qwerltaz committed:
update interval to TimeSpan

qwerltaz committed:
Merge remote-tracking branch 'refs/remotes/upstream/master' into nobody-being-followed-popup

# Conflicts:
#	Content.Server/Ghost/GhostSystem.cs

qwerltaz committed:
Merge remote-tracking branch 'upstream/master' into nobody-being-followed-popup

qwerltaz committed:
dont ask for ghostnado update from client, send from server

server monitors most followed entity and notifies ghosts when it changes. this happens once every 5 seconds.

qwerltaz committed:
disable ghostnado button at start

this is so it doesn't start enabled then immediately blink into disabled after opening the window and then getting update from server.

qwerltaz committed:
Merge remote-tracking branch 'upstream/master' into nobody-being-followed-popup

# Conflicts:
#	Content.Server/Ghost/GhostSystem.cs

qwerltaz committed:
Merge remote-tracking branch 'upstream/master' into nobody-being-followed-popup

# Conflicts:
#	Content.Client/UserInterface/Systems/Ghost/Controls/GhostTargetWindow.xaml
#	Content.Client/UserInterface/Systems/Ghost/Controls/GhostTargetWindow.xaml.cs
#	Content.Client/UserInterface/Systems/Ghost/GhostUIController.cs

Commit:
Merge remote-tracking branch 'upstream/master' into nobody-being-followed-popup

# Conflicts:
#	Content.Shared/Ghost/SharedGhostSystem.cs

Commit:
i solve merg]
